### ISSUE TITLE
Malformed Prometheus metrics

### ISSUE THREAD

**Issue by p3we** [erlio/vernemq#923]
### Environment

- VerneMQ Version: 1.6.0
- Prometheus Version: 2.4.2
- OS: CentoOS 7.5
- Erlang/OTP version (if building from source):

### Expected behavior

Prometheus is able to parse VerneMQ metrics.

### Actual behaviour

Metrics are malformed and cannot be parsed by Prometheus. Below you can find an example error message from Prometheus output:
```
level=warn ts=2018-10-29T11:14:22.657300187Z caller=scrape.go:804 component="scrape manager" scrape_pool=vernemq target=http://127.0.0.1:8888/metrics msg="append failed" err="expected label name, got \"COMMA\""
```

### Additional info
I guess the issue is caused by part of the code responsible for joining labels with commas, it produces invalid fields, like follows:
```
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="not_authorized"} 0
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="bad_username_or_password"} 0
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="server_unavailable"} 0
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="client_identifier_not_valid"} 0
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="unsupported_protocol_version"} 0
# HELP mqtt_connack_sent The number of CONNACK packets sent.
# TYPE mqtt_connack_sent counter
mqtt_connack_sent{node="vernemq@172.16.18.3",mqtt_version="4",,,return_code="success"} 0
```

**Comment by larshesel** [comment]
Thanks for reporting - that sure does look like a bug!

**Comment by larshesel** [comment]
Hi @p3we, I've created a PR in #924 - would you be able to test it?

**Comment by p3we** [comment]
> Hi @p3we, I've created a PR in #924 - would you be able to test it?

Hi @larshesel. Yes, I can confirm that #924 works perfectly. Thanks!
That problem may exists also in 1.5.0, last version that worked for me was 1.4.2. 
Regards.

**Comment by larshesel** [comment]
awesome - thanks! yes, we rewrote the metrics system to support labels in 1.5.0 IIRC, so we should include this fix in 1.5.1 which I think we'll release soon.
